task_sdk/src/airflow/sdk/execution_time/supervisor.py

```diff
@@ -352,7 +341,17 @@ def start(
             del logger
 
             # Run the child entrypoint
-            _fork_main(child_stdin, child_stdout, child_stderr, child_logs.fileno(), target)
+            try:
+                _fork_main(child_stdin, child_stdout, child_stderr, child_logs.fileno(), target)
+            except BaseException as e:
+                try:
+                    print("Exception in _fork_main, exiting with code 124", e, file=sys.stderr)
```

Comment:
Lets log instead?

Reply:
We can't log, not reliably as this is such an unusual occurrence that something went wrong we can't rely on log being usable here

Reply:
I've added a comment about this